lib/slacker/application.rb

```diff
@@ -191,6 +191,7 @@ def query_script_odbc(sql)
         ensure
           st.drop unless st.nil?
       end
+      results.count > 1 ? results : results.first
```

Comment:
In my changed, I introduced a bug not correctly returning the results. Fixed it and tested on Windows 10 with odbc driver.